contracts/truefi2/TrueRateAdjuster.sol

```diff
@@ -263,28 +298,43 @@ contract TrueRateAdjuster is ITrueRateAdjuster, UpgradeableClaimable {
         return ((f64x64Score / MAX_CREDIT_SCORE).fixed64x64Pow(f64x64LimitAdjustmentPower / BASIS_POINTS) * BASIS_POINTS).toUInt();
     }
 
+    /**
+     * @dev Calculate total TVL in USD
+     * @param decimals Precision to return
+     * @return TVL for all pools
+     */
+    function totalTVL(uint8 decimals) public view returns (uint256) {
+        uint256 tvl = 0;
+        uint256 resultPrecision = uint256(10)**decimals;
+
+        // loop through pools and sum tvl accounting for precision
+        for (uint8 i = 0; i < tvlPools.length; i++) {
```

Comment:
I think it should be `uint256 i`.

Reply:
This index was already uint8 when I cut/paste it, but I'm happy to change this to uint256.